src/test/pybind/test_cephfs.py

```diff
@@ -867,8 +823,7 @@ def test_get_pool():
     size=int(s.split(" ")[-1])
     assert_equal(cephfs.get_pool_replication(dp_dict["pool_id"]), size)
 
-@with_setup(setup_test)
-def test_disk_quota_exceeeded_error():
+def test_disk_quota_exceeeded_error(testdir):
```

Comment:
not sure what happened here, but this one is failing in http://qa-proxy.ceph.com/teuthology/cbodley-2023-06-21_14:39:04-fs-main-distro-default-smithi/7309725/teuthology.log:
```
2023-06-21T15:04:07.201 INFO:tasks.workunit.client.0.smithi016.stdout:=================================== FAILURES ===================================
2023-06-21T15:04:07.201 INFO:tasks.workunit.client.0.smithi016.stdout:_______________________ test_disk_quota_exceeeded_error ________________________
2023-06-21T15:04:07.201 INFO:tasks.workunit.client.0.smithi016.stdout:
2023-06-21T15:04:07.201 INFO:tasks.workunit.client.0.smithi016.stdout:testdir = None
2023-06-21T15:04:07.202 INFO:tasks.workunit.client.0.smithi016.stdout:
2023-06-21T15:04:07.202 INFO:tasks.workunit.client.0.smithi016.stdout:    def test_disk_quota_exceeeded_error(testdir):
2023-06-21T15:04:07.202 INFO:tasks.workunit.client.0.smithi016.stdout:        cephfs.mkdir("/dir-1", 0o755)
2023-06-21T15:04:07.202 INFO:tasks.workunit.client.0.smithi016.stdout:        cephfs.setxattr("/dir-1", "ceph.quota.max_bytes", b"4096", 0)
2023-06-21T15:04:07.202 INFO:tasks.workunit.client.0.smithi016.stdout:        fd = cephfs.open(b'/dir-1/file-1', 'w', 0o755)
2023-06-21T15:04:07.202 INFO:tasks.workunit.client.0.smithi016.stdout:        cephfs.ftruncate(fd, 4092)
2023-06-21T15:04:07.202 INFO:tasks.workunit.client.0.smithi016.stdout:        cephfs.lseek(fd, 4090, os.SEEK_SET)
2023-06-21T15:04:07.203 INFO:tasks.workunit.client.0.smithi016.stdout:>       assert_raises(libcephfs.DiskQuotaExceeded, cephfs.write, fd, b"abcdeghiklmnopqrstuvwxyz1234567890qwertyuioddd", -1)

2023-06-21T15:04:07.206 INFO:tasks.workunit.client.0.smithi016.stdout:E           AssertionError: DiskQuotaExceeded not raised
```

Reply:
Might be related to issue fixed by PR #50910. @rishabh-d-dave Could you confirm?

Reply:
thanks, i see that PR adds a `time.sleep(100)` before this check as a workaround